receiver/netflowreceiver/factory.go

```diff
@@ -0,0 +1,52 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package netflowreceiver // import "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/netflowreceiver"
+
+import (
+	"context"
+
+	"go.opentelemetry.io/collector/component"
+	"go.opentelemetry.io/collector/consumer"
+	"go.opentelemetry.io/collector/receiver"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/receiver/netflowreceiver/internal/metadata"
+)
+
+const (
+	defaultSockets   = 1
+	defaultWorkers   = 2
+	defaultQueueSize = 1_000_000
+)
+
+// NewFactory creates a factory for netflow receiver.
+func NewFactory() receiver.Factory {
+	return receiver.NewFactory(
+		metadata.Type,
+		createDefaultConfig,
+		receiver.WithLogs(createLogsReceiver, component.StabilityLevelAlpha))
```

Comment:
```suggestion
		receiver.WithLogs(createLogsReceiver, metadata.LogsStability))
```
We can take this from the mdatagen-created file.